docs/pages/api/access-plugin.mdx

```diff
@@ -0,0 +1,738 @@
+
+Whether creating a new row of the spreadsheet or updating an existing one, we
+need a way to extract data from an Access Reqeust in order to provide it to
+Google Sheets. Below the `googleSheetsPlugin` type declaration, add the
+following function:
+
+```go
+func (g *googleSheetsPlugin) makeRowData(ar types.AccessRequest) *sheets.RowData {
+
+	t := ar.GetCreationTime().String()
+	u := ar.GetUser()
+	s := strings.Join(ar.GetRoles(), ",")
+	i := ar.GetName()
+	a, ok := requestStates[ar.GetState()]
+	l := fmt.Sprintf(
+		"=HYPERLINK(\"%v\", \"%v\")",
+		"https://"+proxyAddr+"/web/requests/"+ar.GetName(),
+		"View Access Request",
+	)
+
+	// Could not find a state, but this is still a valid Access Request
+	if !ok {
+		a = requestStates[types.RequestState_NONE]
+	}
+
+	return &sheets.RowData{
+		Values: []*sheets.CellData{
+			&sheets.CellData{
+				UserEnteredValue: &sheets.ExtendedValue{
+					StringValue: &i,
+				},
+			},
+			&sheets.CellData{
+				UserEnteredValue: &sheets.ExtendedValue{
+					StringValue: &t,
+				},
+			},
+			&sheets.CellData{
+				UserEnteredValue: &sheets.ExtendedValue{
+					StringValue: &u,
+				},
+			},
+			&sheets.CellData{
+				UserEnteredValue: &sheets.ExtendedValue{
+					StringValue: &s,
+				},
+			},
+			&sheets.CellData{
+				UserEnteredValue: &sheets.ExtendedValue{
+					StringValue: &a,
+				},
+			},
+			&sheets.CellData{
+				UserEnteredValue: &sheets.ExtendedValue{
+					FormulaValue: &l,
+				},
+			},
+		},
+	}
+}
+```
+
+`makeRowData` is a method of the `googleSheetsPlugin` type. (The `*` before
+`googleSheetsPlugin` indicates that the method receives a *pointer* to a
+`googleSheetsPlugin`.) It takes a `types.AccessRequest`, which Teleport's API
+library uses to represent the fields within an Access Request.
+
+The Google Sheets client library defines a `sheets.RowData` type that we
+include in requests to update a spreadsheet. This function converts a
+`types.AccessRequest` into a `*sheets.RowData` (another pointer). 
+
+When extracting the data, we use the `types.AccessRequest.GetName()` method to
+retrieve the ID of the Access Request as a string we can include in the
+spreadsheet. 
+
+Users can review an Access Request by visiting a URL within the Teleport Web UI
+that corresponds to the request's ID. `makeRowData` assembles a `=HYPERLINK`
+formula that we can insert into the spreadsheet as a link to this URL.
+
+Next, we'll add two functions that call `makeRowData`, one to create a new row
+and one to update an existing row.
+
+### Create a row
+
+Below `makeRowData`, add the following:
+
+```go
+func (g *googleSheetsPlugin) createRow(ar types.AccessRequest) error {
+
+	row := g.makeRowData(ar)
+
+	req := sheets.BatchUpdateSpreadsheetRequest{
+		Requests: []*sheets.Request{
+			{
+				AppendCells: &sheets.AppendCellsRequest{
+
+					Fields: "*",
+					Rows: []*sheets.RowData{
+						row,
+					},
+				},
+			},
+		},
+	}
+
+	resp, err := g.sheetsClient.BatchUpdate(spreadSheetID, &req).Do()
+
```

Comment:
Whitespace is a great way to separate logical pieces of code. In this case, though, the error handling should be considered part of the same task.
```suggestion
```